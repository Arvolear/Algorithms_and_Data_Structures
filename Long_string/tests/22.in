531052756980
97209429295