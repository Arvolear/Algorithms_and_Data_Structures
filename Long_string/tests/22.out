628262186275
433843327685
51623335431562127729100
5